common/Graphics Pipeline Details/Color Combiner.md

```diff
@@ -0,0 +1,171 @@
+# Color Combiner
+
+In one and two cycle mode, the color combiner takes input from the texturing unit. The color combiner is fed various inputs, and combines them via user set inputs. Pixel output from the color combiner is fed into the chroma key & alpha fix up unit and then into the alpha compare unit.
+
+## Inputs
+
+The color combiner has inputs a, b, c, d for color and alpha separately.
+See [f3d_materials](F3D Material Overview.md) for some explanation on various inputs.
```

Comment:
This won´t link correctly

Reply:
bump

Reply:
yeah I'm not sure how to fix it I'll have to look at build logs and do testing